src/utils/datetime/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss'
```

Comment:
This is a temporary duplicate. We will replace all instances in our codebase with this one.